Mathlib/Order/SuccPred/Basic.lean

```diff
@@ -445,15 +451,11 @@ theorem le_le_succ_iff : a ≤ b ∧ b ≤ succ a ↔ b = a ∨ b = succ a := by
   · exact ⟨le_succ a, le_rfl⟩
 #align order.le_le_succ_iff Order.le_le_succ_iff
 
-theorem _root_.CovBy.succ_eq (h : a ⋖ b) : succ a = b :=
-  (succ_le_of_lt h.lt).eq_of_not_lt fun h' => h.2 (lt_succ_of_not_isMax h.lt.not_isMax) h'
-#align covby.succ_eq CovBy.succ_eq
+/-- See also `Order.le_succ_of_wcovBy`. -/
+lemma succ_eq_of_wcovBy (h : a ⋖ b) : succ a = b := (succ_le_of_lt h.lt).antisymm h.wcovBy.le_succ
```

Comment:
Shouldn't this be `of_covBy`?

Reply:
I'm not on my computer (for a while) so you could please fix it yourself/suggest the change in a comment?